docs/agents/sematext-agent/autodisco/mysql-driver.md

```diff
@@ -0,0 +1,8 @@
+title: MySQL driver
+description:  Copying the MySQL driver into the agent volumes
+
+Due to license restrictions, the MySQL Connector is not bundled within the `sematext/app-agent` Docker image. Instead, Sematext Agent automatically downloads the MySQL jar artifact from the Maven repository according to the version of MySQL service that was discovered.
```

Comment:
... Maven repository according to the version of MySQL service that was discovered. => ... Maven repository according to the version of the MySQL service that was discovered.